cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -29,9 +29,9 @@ end
 parameter "param_service" do
   category "RI"
   label "Service"
-  default "All"
+  default "All Except EC2"
```

Comment:
Changing default value for an existing param is a breaking change.  Also, should this new value be the default?

Reply:
answered in the below comment.